tests/testthat/_snaps/sparse-glmnet.md

```diff
@@ -2,7 +2,8 @@
 
     Code
       mold(rec, mlc_churn, blueprint = sparse_bp)
-    Error <rlang_error>
-      `data` must only contain numeric columns.
+    Condition
+      Error in `recompose()`:
```

Comment:
Is this a regression? Not sure if this is a change in rlang or our own, but if this seems like a step backward I say we file an issue.

Reply:
edit: referring to the reference to `recompose()` newly showing up.